apple/internal/rule_attrs.bzl

```diff
@@ -571,20 +571,6 @@ of the target will be used instead.
 The desired name of the executable, if the bundle has an executable. If this attribute is not set,
 then the name of the `bundle_name` attribute will be used if it is set; if not, then the name of
 the target will be used instead.
-""",
-        ),
-        # TODO(b/36512239): Rename to "bundle_post_processor".
-        "ipa_post_processor": attr.label(
-            allow_files = True,
-            executable = True,
-            cfg = "exec",
-            doc = """
-A tool that edits this target's archive after it is assembled but before it is signed. The tool is
-invoked with a single command-line argument that denotes the path to a directory containing the
-unzipped contents of the archive; this target's bundle will be the directory's only contents.
-
-Any changes made by the tool must be made in this directory, and the tool's execution must be
-hermetic given these inputs to ensure that the result can be safely cached.
```

Comment:
Doesn't this change actually remove the attribute for rules that could set it? I think "for rules that aren't using it" means inside of Google, but we probably want to add `rule_attrs.ipa_post_processor_attrs(),` back to all the rules that previously had it?

Reply:
I see that the attr was kept specifically in only a few rules now (main bundling rules such as `ios_application`, `ios_extension`, `macos_command_line_application`, etc.). I do wonder how many people use this feature in rules like `ios_static_framework` today for example and if it's worth keeping, but I think you're right, the reason behind the change was likely a clean up at Google. If we want to be consistent and keep it everywhere it's supported today, then yes, we shouldn't take this change.

Reply:
> then yes, we shouldn't take this change

Well, I think we should take the change (because of the `def _ipa_post_processor_attrs()` refactoring, which really should have been it's own commit IMO), but add `rule_attrs.ipa_post_processor_attrs(),` to all the other bundle rules.